guides/working-with-git.md

```diff
@@ -2,10 +2,11 @@
 
 ## Committing Code
 
-Follow 3 simple rules:
+Follow these simple steps:
 
-1. Commit locally to the _working_ branch as often as you save.
+1. Commit often. - As _often_ as you change a pice of code that is testable and compiles.
```

Comment:
```suggestion
1. Commit often. - _Whenever_ you change a piece of code that is testable and compiles.
```
I don't know if this still conveys the same message that you were trying to pass. I just thought it was a bit more direct.